doc/src/read_data.rst

```diff
@@ -618,7 +618,7 @@ of analysis.
    * - tdpd
      - atom-ID atom-type x y z cc1 cc2 ... ccNspecies
    * - template
-     - atom-ID molecule-ID template-index template-atom atom-type x y z
+     - atom-ID atom-type x y z molecule-ID template-index template-atom
```

Comment:
Not clear why this was re-ordered.  I think the old ordering was correct, and matches atom_vec_template.cpp fields_data_atom.

Reply:
Will need to review this. This may be an unexpected side-effect of having use atom style hybrid to include support for charges.

Reply:
this has been restored and confirmed with examples